jerry-core/ecma/base/ecma-globals.h

```diff
@@ -90,6 +90,11 @@ typedef enum
  */
 typedef uint32_t ecma_value_t;
 
+/**
+ * Type for small integer numbers in JerryScript.
+ */
+typedef int32_t ecma_value_integer_t;
```

Comment:
Terminology: I think `ecma_integer_t` or `ecma_integer_value_t` would be closer to the naming schemes we are already using.